src/Tokenizer/Tokens.php

```diff
@@ -1088,16 +1088,18 @@ public function clearRange(int $indexStart, int $indexEnd): void
      */
     public function isMonolithicPhp(): bool
     {
-        $size = $this->count();
-
-        if (0 === $size) {
+        if (0 === $this->count()) {
             return false;
         }
 
-        if ($this->isTokenKindFound(T_INLINE_HTML)) {
+        if ($this->countTokenKind(T_INLINE_HTML) > 1) {
             return false;
         }
 
+        if (1 === $this->countTokenKind(T_INLINE_HTML)) {
+            return 1 === Preg::match('/^#!.*$/m', $this[0]->getContent());
+        }
+
         return 1 >= ($this->countTokenKind(T_OPEN_TAG) + $this->countTokenKind(T_OPEN_TAG_WITH_ECHO));
```

Comment:
```suggestion
        return 1 === ($this->countTokenKind(T_OPEN_TAG) + $this->countTokenKind(T_OPEN_TAG_WITH_ECHO));
```

do we have a case when we may have no T_OPEN_TAG, no T_OPEN_TAG_WITH_ECHO, file is non-empty, and we still claim it's monolithic?

Reply:
Apparently no, and we never had.